games-board/goatee-gtk/goatee-gtk-0.4.0.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="A monadic take on a 2,500-year-old board game - GTK+ UI"
+HOMEPAGE="https://khumba.net/projects/goatee"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="AGPL-3"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=dev-games/goatee-0.4:=[profile?] <dev-games/goatee-0.5:=[profile?]
```

Comment:
I don't love doing these on the same line _but_ if it's the Haskell style, that's fine. We let e.g. the Perl project have their own style wrt sorting in a specific way too.

Reply:
I've had some conversations with @trofi about this, see [here][1] and [here][2], and I think I have some IRC logs as well. Long story short, it's more-or-less as you've said - it's the "Haskell style".

Personally, I prefer one per line as well, but rather that fight that battle I prefer to focus my energies trying to revitalize [bug 4315][3].  I think @mgorny does a good job [laying out the problem][4] and discussing why such a feature is necessary.

[1]: https://github.com/gentoo-haskell/gentoo-haskell/pull/1097
[2]: https://github.com/gentoo-haskell/gentoo-haskell/pull/1099#discussion_r546277694
[3]: https://bugs.gentoo.org/4315
[4]: https://wiki.gentoo.org/wiki/Future_EAPI/Version_syntax_changes#Version_ranges

Reply:
💯